app/[locale]/not-found.tsx

```diff
@@ -17,6 +18,15 @@ const NotFoundPage: FC = () => {
       <main>
         404
         <h1 className="special -mt-4">{t('layouts.error.notFound.title')}</h1>
+        <div className="my-4 flex items-center justify-center">
+          <Image
+            src="/static/images/node-mascot.svg"
+            alt="The Node.js mascot"
+            className="w-[100px]"
+            height={200}
+            width={174.375}
```

Comment:
```suggestion
            height={114.7}
            width={100}
```

We can directly give the width we want according to the aspect ratio

Reply:
curious, why is this better than giving it a height based on the aspect ratio? because of the TW class?

Reply:
AFAIK When we add the `w-[100px]` class, the width and height values we give to the image element no longer matter. The width of the rendered image with becomes `100px` and the height becomes `114.69px`.

![image](https://github.com/nodejs/nodejs.org/assets/14062599/939e2e6a-5cba-430d-9d94-de69197daec0)

In addition to seeing the given input directly, it also helps the `next/image` component to prepare a more optimized `srcset` for us. In this scenario, it dont looks make any sense since we are using svg, but it is quite useful under normal conditions.

Example srcset for `100x114.69`;
`srcset="/_next/image?url=%2Fstatic%2Fimages%2Fnode-mascot.svg&w=128&q=75 1x, /_next/image?url=%2Fstatic%2Fimages%2Fnode-mascot.svg&w=256&q=75 2x"`

Example srcset for `174.375x200`;
`srcset="/_next/image?url=%2Fstatic%2Fimages%2Fnode-mascot.svg&w=256&q=75 1x, /_next/image?url=%2Fstatic%2Fimages%2Fnode-mascot.svg&w=384&q=75 2x"`